tavern/ent/schema/tome.go

```diff
@@ -0,0 +1,63 @@
+package schema
+
+import (
+	"time"
+
+	"entgo.io/contrib/entgql"
+	"entgo.io/ent"
+	"entgo.io/ent/schema/field"
+)
+
+// Tome holds the schema definition for the Tome entity.
+type Tome struct {
+	ent.Schema
+}
+
+// Fields of the Tome.
```

Comment:
Does this describe the tome only on disk or as it's being passed to an agent as well?

Reply:
Stores backend info, it will differ both from how it is stored on disk as well as from how it is passed to an agent (probably)